<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>751</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Таблица распределённых студентов</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>721</width>
      <height>491</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Преподаватель</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Студент</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>590</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1146</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
